video_xblock/backends/wistia.py

```diff
@@ -209,10 +206,7 @@ def get_default_transcripts(self, **kwargs):
 
     @staticmethod
     def format_transcript_text_line(line):
-        """
-        Replaces comma with dot in timings, e.g. 00:00:10,500 should be 00:00:10.500
-
-        """
+        """Replace comma with dot in timings, e.g. 00:00:10,500 should be 00:00:10.500."""
         pattern = re.compile(r"\d{2}:\d{2}:\d{2},\d{3}")
```

Comment:
This is out of scope of this PR. But why this method can't be simplified down to something like this?
```py
return re.sub(r'(\d{2}:\d{2}),(\d{3})', r'\1.\2', line)
```